src/osd/scrubber/scrub_backend.cc

```diff
@@ -1282,7 +1286,9 @@ bool ScrubBackend::compare_obj_details(pg_shard_t auth_shard,
     auth_bl.push_back(auth_attr->second);
 
     if (!can_bl.contents_equal(auth_bl)) {
-      format_to(std::back_inserter(out), "{}object info inconsistent ", sep(error));
+      format_to(std::back_inserter(out),
+                "{}object info inconsistent ",
```

Comment:
Space between {} and object

Reply:
Also, unnecessary change?

Reply:
Adding the space will change existing logs. Might affect tests. Not part of this PR.
Re the formatting - a result of forcing  80-cols. Will move elsewhere. See the general comment below.